app/controllers/users_controller.rb

```diff
@@ -10,6 +10,7 @@ def index
   def create
     @user = User.new(user_params)
     if @user.save
+      session[:user_id] = @user.id
```

Comment:
this allows the new registered user's information to be stored in the session so that they become the current user